tier4/{{cookiecutter.project_slug}}/.github/codejson/{{cookiecutter.project_name}}/code.json

```diff
@@ -33,6 +33,8 @@
   "localisation": "{{ cookiecutter.localisation }}",
   "repoType": "{{ cookiecutter.repo_type }}",
   "userInput": "{{ cookiecutter.user_input }}",
+  "usageType": "{{ cookiecutter.usage_type }}",
+  "exemption": "{{ cookiecutter.exemption }}",
```

Comment:
As per the code.json schema, could `exemption` be changed to `exemptionText`? Also update accordingly in `cookiecutter.json`

Reply:
https://github.com/GSA/code-gov-data/blob/master/schemas/schema-2.0.0.json#L97